src/ConnectionString.php

```diff
@@ -164,4 +164,14 @@ public static function getUriFromConnectionString(string $connectionString): ?Ur
 
         return null;
     }
+
+    /**
+     * @param string $connectionString
+     *
+     * @return string[]
+     */
+    private static function getParts(string $connectionString): array
+    {
+        return \array_map('\trim', \explode(';', $connectionString));
```

Comment:
The trim is because spaces may occur in the connection string:

https://eventstore.org/docs/dotnet-api/connecting-to-a-server/index.html#connection-string

However I think not all cases are supported yet, like this one:

```
"Connect To = tcp://admin:changeit@localhost:1113; Gossip Timeout = 500"
```